docs/UsersGuide/source/ConfigWorkflow.rst

```diff
@@ -86,6 +86,8 @@ Directory Parameters
 ``EXPT_BASEDIR``: (Default: “”)
    The base directory in which the experiment directory will be created.  If this is not specified or if it is set to an empty string, it will default to ``${HOMErrfs}/../expt_dirs``, where ``${HOMErrfs}`` contains the full path to the ``regional_workflow`` directory.
 
+NOTE DOM 20220227 need to double check, beforehamnd expt_dirs sits in ufs-srweather-app/../expt_dirs, i.e. homerrfs would be ufs-srweather-app
+
 ``EXPT_SUBDIR``: (Default: “”)
```

Comment:
${HOMErrfs}=ufs-srweather-app/regional_workflow, therefore need to change ${HOMErrfs}/../expt_dirs to ${HOMErrfs}/../../expt_dirs.

Reply:
Fixed.